index.rst

```diff
@@ -52,163 +56,110 @@ The distribution of the HiTS survey fields in the sky is shown in :ref:`Figure 2
-
-.. code-block:: python
-
-   repo = 'ingested_15A38/'  # used by ingest, ingestCalibs, processCcd
-   calibrepo = 'calibingested_15A38/'  # used by ingestCalibs, processCcd
-   processedrepo = 'processed_15A38/'  # used by processCcd, diffIm
-   diffimrepo = 'diffim_15A38_g/'  # used by diffIm
-   visits = [410927, 411033]  # used by processCcd, diffIm
-   # NOTE: visits assumes the first element is template and the rest are science
-   ccdnum = '1..62'  # used by processCcd, diffIm
-   # NOTE: the default '1..62' value includes all of the DECam CCDs
-
-In general, ``repo`` refers to where ingested images will live, ``calibrepo``
-refers to where ingested calibration products (flats and biases) will live, ``processedrepo`` refers to where "calexp"
-images will live (i.e., those that have been processed with ``processCcd`` including steps 1 through 3 in :ref:`Overview <Overview>`),
-and ``diffimrepo`` refers to where difference images and DIA Sources (catalogs) will ultimately live.
-These should each be different directories, and it's recommended to have them all reside in the same top-level directory.
-Visit numbers can be found in image headers or retrieved from the registry database created in ``repo`` during image ingestion 
-(visit numbers are not used during ingestion, so you may set them after this step). The first visit you specify
-in the list will be used as the template.
-
-Finally, copy or link the Pan-STARRS astrometric reference catalog into the directory you've chosen for 
-``repo`` and call it ``ref_cats``. If you are working on the ``lsst-dev`` server, you can link the Gaia, 
-Pan-STARRS, and SDSS catalogs by 
+You may set all of this up as follows:
 
 .. prompt:: bash
-   
-   mkdir repo
-   ln -s /datasets/refcats/htm/htm_baseline repo/ref_cats
-
-Note that the ``repo`` directory is called ``ingested_15A38`` in the default values given above.
-If you wish to use an astrometric reference catalog other than Pan-STARRS, you must update the code in the ``doProcessCcd``
-function of ``decam_process.py`` accordingly. It is not necessary to explicitly ``mkdir`` the other repositories.
-
-If you really don't want to deal with astrometric reference catalogs, you can skip the astrometry and 
-photometric calibration steps by editing the contents of ``args`` in the ``doProcessCcd`` function of ``decam_process.py``. 
-In this situation, you would set both ``calibrate.doAstrometry=False`` and ``calibrate.doPhotoCal=False``. 
-Be aware that the difference imaging will not work well in this case, however, because the visits 
-will not be precisely lined up to the same WCS.
 
-Once you are ready, run the following:
+   setup lsst_apps
+   git clone https://github.com/lsst/obs_decam.git
+   git clone https://github.com/lsst-dm/ap_pipe.git
+   setup -k -r obs_decam
+   setup -k -r ap_pipe
+   git clone https://github.com/lsst/ap_verify_hits2015.git
```

Comment:
Is something like this really inside the scope of the tech note? I would think the target audience already has their own way of setting up the Stack.